### ISSUE TITLE
[CoreData Kit] Second level relationship causes crash

### ISSUE THREAD

**Issue by iGranDav** [iSofTom/STDebugKit#2]
When navigating on my core data objects, and showing a second level relationship entity. This always causes a crash.

`-[__NSCFString isTemporaryID]: unrecognized selector sent to instance` in `STDebugKitModuleCoreData.m` line 89.

**Comment by iSofTom** [comment]
Right, because you shouldn't be able (yet) to navigate to a second level via a to-many relationship. This will be fixed in the next version.
